x-pack/plugins/maps/public/angular/map_controller.js

```diff
@@ -6,7 +6,7 @@
 
 import chrome from 'ui/chrome';
 import React from 'react';
-import { I18nContext } from 'ui/i18n';
+import { I18nProvider } from '@kbn/i18n/react';
```

Comment:
Forgot to translate following
* breadcrumbs `Maps`
* Saving error messages and toast messages
* top nav button labels and description text - full screen, inspector, and save